pymessenger/bot.py

```diff
@@ -368,3 +368,57 @@ def remove_persistent_menu(self):
             )
             result = response.json()
             return result
+    def send_text_quick_replies(self, recipient_id, listOfReplies, listOfPayloads=[], message=""):
+        """
+        Sends a list of text quick replies with an optional message to send before sending the quick replies
+        Payload and Message are optional, however, if no payload for a specific reply (i.e. None) 
+        or for all quick replies, the payload will be defined as the reply itself.
+        
+        
+        https://developers.facebook.com/docs/messenger-platform/send-messages/quick-replies/#text
+        
+        Output:
+            Response from API as <dict>
+        """
+        quickRepliesList = []
+        
+        # If no payloads identified
+        if len(listOfPayloads) == 0:     
+            for reply in listOfReplies:
+                quickRepliesList.append({
+                                        "content_type":"text",
+                                        "title":reply,
+                                        "payload":reply
+                                        })
+            
+        # If payloads is identified
+        else:
+            for reply, payload in zip(listOfReplies, listOfPayloads):
+                # if some payload is not identified in the list
+                if payload == None:
+                    quickRepliesList.append({
+                                            "content_type":"text",
+                                            "title":reply,
+                                            "payload":reply
+                                            })
+                if payload != None:
```

Comment:
Couldn't this just be an `else`?

Reply:
That's true. I changed it to an else. Thanks for your suggestion!